pkg/deploy/lattice/target_group_synthesizer.go

```diff
@@ -70,7 +70,7 @@ func (t *targetGroupSynthesizer) SynthesizeTriggeredTargetGroup(ctx context.Cont
 	for _, resTargetGroup := range resTargetGroups {
 
 		// find out VPC for service import
-		if resTargetGroup.Spec.Config.IsServiceImport {
+		if resTargetGroup.Spec.Config.IsServiceImport || resTargetGroup.Spec.Config.IsServiceExport {
 			/* right now, TG are unique across VPC, we do NOT need to get VPC
```

Comment:
when handling `Kind: ServiceExport` k8s resource, the latticestore should create a `IsServiceImport == true` cache for it, so that when httproute_controller.go's buildRules() synthesizing Listener's rules, synthesizer could retrieve this record from latticestore for this ServiceImport backendRefs. https://github.com/aws/aws-application-networking-k8s/blob/03157875c7d9652af5c5ee354bd4aa775f1b75c2/pkg/deploy/lattice/rule_manager.go#L186

if line 102 return err: `tgStatus, err := t.targetGroupManager.Get(ctx, resTargetGroup)`, if means this  ServiceExport refered tg haven't yet been created by a `Kind:Service`, it just don't create this record in the latticeStore, and trying to create it in the next run retrying


TODO: Not sure do we need to change any delete logic for it?

Reply:
Ok, got it I will rollback this change.

In that case, should I use this way to test service import:
1. invoke a CP api to create a ip type target group
2. create a serviceimport it could identify this created target group

Also, does the purpose of serviceExport is to make a serviceImport could find that target group? what the purpose of serviceExport?